blathering-barnacles/Bangazon-3
# Add back in missing fields on employee serializer

## Link to Ticket
No ticket

## Description of Proposed Changes
I forgot to add back in some of the fields for employee when testing things out while working on my ticket.
- 

## Steps to Test

Outline the steps to test

```sh
git fetch --all
git checkout jb-final-testing
```
Take a look at the employee resource and make sure you can see all the employee fields.

## Impacted Areas in Application

List general components of the application that this PR will affect:

* Employee resource
